woocommerce/woocommerce-gutenberg-products-block
# Auto-insert the Mini Cart block after the Navigation block

Fixes https://github.com/woocommerce/woocommerce/issues/42385

## Why



Following up on the work made on https://github.com/WordPress/gutenberg/pull/51449,  the Mini Cart block can now be auto-inserted right after the Navigation block.

Since this is still an experimental feature within Gutenberg, we are here restricting its usage to the feature plugin at this first moment.

## Testing Instructions



1. Make sure you have the latest version of the Gutenberg plugin installed and enabled
2. Under Gutenberg > Experiments, enable the auto-inserting feature:
<img width="639" alt="Screenshot 2023-08-17 at 15 53 24" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/ffaa413d-f94c-40f8-8c69-c6ded28f437a">

3. Now head over to Appearance > Editor > Patterns > Header
4. If your header is customized, clear all customizations:

<img width="1012" alt="Screenshot 2023-08-17 at 15 57 19" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/0ed297de-1043-4aa0-92cf-7b74bd8d63f7">

5. Make sure you can see the mini-cart block in the navigation preview:

<img width="393" alt="Screenshot 2023-08-17 at 15 59 05" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/96492633-32f4-47c0-b687-e7f2a3944336">

7. Start editing the header and make sure you can see the mini-cart in the editor:

<img width="1241" alt="Screenshot 2023-08-17 at 15 59 27" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/fddea3f6-f01f-47a5-b4ea-7ea1331d3031">

8. Access your site on the front-end and make sure you can see the mini-cart block rendered right after the navigation block as well:

<img width="1228" alt="Screenshot 2023-08-17 at 15 59 44" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/d6d0d5cf-9dd6-4929-b279-9cb6b8d8d318">


## Screenshots or screencast



| Before | After |
| ------ | ----- |
| <img width="1228" alt="Screenshot 2023-08-17 at 16 07 19" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/74dbd990-c2cf-4d1a-8c69-55af9954e0eb"> | <img width="1228" alt="Screenshot 2023-08-17 at 15 59 44" src="https://github.com/woocommerce/woocommerce-blocks/assets/15730971/06c01756-0a92-469e-a077-5214a0344e32"> |

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Enable auto-insert for the Mini Cart block after the Navigation block.
